apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './index-table';
-export { CellRenderer, HeadingLabel } from './types';
+export * from './types';
```

Comment:
Did this work?  What's the difference?

Reply:
I suspect (but don't want to waste time) that maybe we're importing from the nested module instead of here and the unused modules somehow get dropped in the build in such a way that the wildcard still works but explicit imports.

The point of doing the export that way was that I didn't want to automatically expose everything from types. But the index file there is the one that should spell this out, so it was silly to do this.

Moral of the story: don't do it this way.